src/myfunctions.js

```diff
@@ -0,0 +1,100 @@
+import {array} from './index.js'; // eslint-disable-line
+
+const todoList = () => {
```

Comment:
- [ ] Your app produces an error, please consider fixing it so your app works correctly 

![Screenshot_20220610_172819](https://user-images.githubusercontent.com/25227022/173113055-1210b0d7-a1ad-4129-9dd6-c360ad96e333.png)